lib/ilp.js

```diff
@@ -100,12 +100,6 @@ class Ilp {
 
     this.log.info('Closing connection to ILP connector...')
     await connection.end()
-
-    // connection will probably disconnect the plugin automatically
-    if (plugin.isConnected()) {
```

Comment:
why did you remove this?

Reply:
Oh ya I meant to mention that in the PR desc. I confirmed the "probably" -- the connection auto closes the plugin on end.